src/main/java/stringcalculator/Calculator.java

```diff
@@ -0,0 +1,32 @@
+package stringcalculator;
+
+import java.util.regex.Pattern;
+
+public class Calculator {
+
+    public static final int DEFAULT_RESULT = 0;
+    public static final String CUSTOM_REGEX = "//.\\n";
+    static CalculatorNumbers calculatorNumbers;
```

Comment:
상수와 상태 값은 줄바꿈을 통해서 구분하기 쉽게해주시면 좋을 것 같아요. 😃

```suggestion
    public static final String CUSTOM_REGEX = "//.\\n";
    
    static CalculatorNumbers calculatorNumbers;
```

Reply:
상수와 상태값 분리 컨벤션 말씀 감사합니다
수정했씁니다